### ISSUE TITLE
please delete

### ISSUE THREAD

**Issue by David-Hari** [huggingface/diffusers#1852]
Please close this, I meant to report this to a forked project but it went here instead.

**Comment by David-Hari** [comment]
Perhaps the problem of blank image is due to the loss going to nan at some point during training. I'm not sure what causes that, perhaps the learning rate is too high. When I reduced to 1e6 I was able to get at least 100 steps, but eventually it still does to nan.
[Here](https://pastebin.com/vUJbGQQB) is a brief log of the `model_pred` and `target` tensors at the line
`loss = F.mse_loss(model_pred.float(), target.float(), reduction="mean")`
